src/main/java/me/crespel/runtastic/parser/SportSessionParser.java

```diff
@@ -6,26 +6,45 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import org.apache.commons.io.FilenameUtils;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import com.topografix.gpx._1._1.GpxType;
+
 import me.crespel.runtastic.model.ElevationData;
 import me.crespel.runtastic.model.GpsData;
 import me.crespel.runtastic.model.HeartRateData;
+import me.crespel.runtastic.model.ImagesMetaData;
+import me.crespel.runtastic.model.Shoe;
 import me.crespel.runtastic.model.SportSession;
+import me.crespel.runtastic.model.SportSessionAlbums;
+import me.crespel.runtastic.model.User;
 
 /**
  * Sport session parser.
  * This class reads sport sessions and related data exported as JSON.
  * @author Fabien CRESPEL (fabien@crespel.net)
+ * @author Christian IMFELD (imfeldc@gmail.com)
  */
 public class SportSessionParser {
 
 	public static final String ELEVATION_DATA_DIR = "Elevation-data";
 	public static final String GPS_DATA_DIR = "GPS-data";
 	public static final String HEARTRATE_DATA_DIR = "Heart-rate-data";
+	public static final String PHOTOS_META_DATA_DIR = "Photos\\Images-meta-data";
+	public static final String PHOTOS_SPORT_SESSION_ALBUMS_DIR = "Photos\\Images-meta-data\\Sport-session-albums";

```

Comment:
Don't use Windows separators (`\\`), prefer adding `File.separator`.
Also, there is some code duplication with ExportConverter, consider moving all static constants to a dedicated ExportConstants class or interface.

Reply:
Thanks! Use File.separator. Will be part of next delivery.

Reply:
Open https://github.com/fcrespel/runtastic-export-converter/issues/5 to track further changes.